third_party/boringssl/add_boringssl_s390x.patch

```diff
@@ -1,17 +1,6 @@
-diff -ur a/BUILD b/BUILD
---- a/BUILD	2017-10-10 15:50:34.000000000 +0000
-+++ b/BUILD	2017-10-15 21:19:02.057606476 +0000
-@@ -63,6 +63,7 @@
-     "-Wwrite-strings",
-     "-Wshadow",
-     "-fno-common",
-+    "-Wno-uninitialized",
- 
-     # Modern build environments should be able to set this to use atomic
-     # operations for reference counting rather than locks. However, it's
 diff -ur a/src/include/openssl/base.h b/src/include/openssl/base.h
---- a/src/include/openssl/base.h	2017-10-10 15:50:34.000000000 +0000
-+++ b/src/include/openssl/base.h	2017-10-15 19:49:38.182154627 +0000
+--- a/src/include/openssl/base.h	2017-10-17 21:30:48.000000000 +0000
++++ b/src/include/openssl/base.h	2017-10-18 04:07:23.590036926 +0000
```

Comment:
Existing issue, but per the discussion here, you probably don't want to be patching this at all anymore.
https://github.com/tensorflow/tensorflow/pull/11170#issuecomment-318697044

Reply:
Thanks @davidben for pointing out. The patch has been removed.